### PR TITLE
Error Pages UI

### PR DESCRIPTION
**Pull request by jindaln**
Added UI for the error pages and added error handling for user not entering nickname

### CODE REVIEW

File: src/main/webapp/errors/invalidAuthRep.html [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <div class="container">
+        <h2>Invalid Authorization</h2>
+        <p> Sorry, your username or password is incorrect </p>
+        <a href="/loginRep.html"> Return to login </a>
+    </div>
+</body>
+</html>
```
Comment (swetha-gangu) [inline src/main/webapp/errors/invalidAuthRep.html:18]: new line

Comment (swetha-gangu) [inline src/main/webapp/errors/invalidAuthRep.html:18]: new line

File: src/main/java/com/google/sps/servlets/RepSubmitQuestionnaireServlet.java [hunk 2]
```diff
@@ -51,9 +51,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         topicList.replaceAll(s -> repName.replaceAll("\\s+","") + s);
         String platforms = request.getParameter(PLATFORM_LIST);
         List<String> platformList = Arrays.asList(platforms.split("\\*,"));
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/RepSubmitQuestionnaireServlet.java:53]: comment?

File: src/main/webapp/errors/invalidAuthRep.html (hunk 1 above)
Comment (v1shanker) [reply src/main/webapp/errors/invalidAuthRep.html:18]: +1 trailing new line

**Pull request merged**